dev-support/Jenkinsfile

```diff
@@ -628,6 +630,118 @@ pipeline {
             }
           }
         }
+
+        stage ('yetus jdk17 hadoop3 checks') {
```

Comment:
For nightly jobs we usually align the files for all branches, so do we want to add this step for all branches?

Reply:
I can do the same for all the active branches.

Reply:
I just mean whether we should add a 'when' section to only run this step for some branches.

At least I tried to run UTs with JDK17 on branch-2.5, there are failures...

Reply:
I think folks are running 2.4+ with JDK17, so we should look into the failures.

I think that you can keep it pinned to 3.x+ for now, allow you to finish rolling out the feature, then go back and update for branches-2.x as a separate task.

Reply:
Currently added master and branch-3 for now. Once it's fine and fixing all the issues if any then backport to 2.x branches.